src/templates/drizzle/labels.hbs

```diff
@@ -0,0 +1,11 @@
+{{#if data.labels}}
```

Comment:
I would keep the logic out of this partial and include in each parent if needed. The partial should assume that `labels` is present. 

it can be called like:

```hbs
{{#if data.labels}}
  {{> drizzle.labels labels=data.labels}}
{{/if}}
```

And then the partial can iterate over `labels`:

```hbs
{{#each labels as |label|}}
...
{{/each}}
```